### PR TITLE
Deploy images with exact version (not only latest/build number)

### LINKED ISSUE #74: Move to versioned docker images
Currently, we're publishing only latest tag for the API. Need also to publish images with versioned tags.

### PR DESCRIPTION
**Pull request by omerlh**
close #74

### CODE REVIEW

File: src/decrypt-api/Startup.cs [hunk 1]
```diff
@@ -80,6 +80,7 @@ public Startup(IHostingEnvironment env)
                             Configuration.GetValue<int>("KeyManagement:KeyVault:MaximumDataLength"));
                     case "AESKey":
                         var key = Configuration.GetValue<string>("KeyManagement:AES:Key");
+                        Console.WriteLine($"AES key used: {key}");
```
Comment (shaikatz) [inline src/decrypt-api/Startup.cs]: Why you don't use the logger?

File: src/encrypt-api/Startup.cs [hunk 2]
```diff
@@ -28,15 +29,16 @@ public Startup(IHostingEnvironment env)
...
                 .SetBasePath(env.ContentRootPath)
                 .AddJsonFile(appsettingsPath, optional: true, reloadOnChange: true)
                 .AddJsonFile("secrets/appsettings.secrets.json", optional: true)
                 .AddEnvironmentVariables();
 
-            Configuration = builder.Build();
+            Configuration = builder.Build();
+
+            var version = Assembly.GetExecutingAssembly().GetName().Version.ToString();
+            Console.WriteLine($"Kamus Encryptor API {version} starting");
```
Comment (shaikatz) [inline src/encrypt-api/Startup.cs]: Why you don't use the logger?

File: src/decrypt-api/Startup.cs (hunk 1 above)
Comment (omerlh) [reply src/decrypt-api/Startup.cs]: I think this is called before the logger is initialized

File: src/encrypt-api/Startup.cs (hunk 2 above)
Comment (omerlh) [reply src/encrypt-api/Startup.cs]: Same as above...

File: src/decrypt-api/Startup.cs (hunk 1 above)
Comment (shaikatz) [inline src/decrypt-api/Startup.cs]:
```suggestion
```

**Pull request merged**